src/charting/EChart.js

```diff
@@ -25,6 +25,14 @@ export default class EChart extends React.Component {
         this.echart.setOption(nextProps.options);
     }
 
+    componentDidUpdate(nextProps) {
+        if (nextProps.style !== this.props.style) {
+            this.echart.resize();
+        }
+
+        this.echart.setOption(nextProps.options);
```

Comment:
Yes, opps, I'll remove changes in this file, 
I was fiddling around to try fix the resizing yesterday, maybe I revert the changes to a wrong commit